packages/ckeditor5-link/src/linkui.ts

```diff
@@ -1185,6 +1260,28 @@ export default class LinkUI extends Plugin {
 		return extractTextFromLinkRange( linkRange );
 	}
 
+	/**
+	 * Returns a provider by its URL.
+	 *
+	 * @param href URL of the link.
+	 * @returns Link provider and item or `null` if not found.
+	 */
+	private _getLinkProviderLinkByHref( href: string ): { provider: LinksProvider; item: LinksProviderItem } | null {
+		if ( !href ) {
+			return null;
+		}
+
+		for ( const provider of this._linksProviders ) {
+			const item = provider.getItems().find( item => item.href === href );
```

Comment:
```suggestion
			const item = provider.getItem( href );
```

Reply:
@niegowski fixed in https://github.com/ckeditor/ckeditor5/pull/17593/commits/d4eaf64a2ecd1bf99c7670dd6604449a62a8cfe7